migration/readme.md

```diff
@@ -0,0 +1,17 @@
+### For Installation Guide refer:
+- https://github.com/golang-migrate/migrate/blob/master/cmd/migrate/README.md
+
+### For best practices refer: 
+- https://github.com/golang-migrate/migrate/blob/master/MIGRATIONS.md
+
+### Facing a dirty migration issue? Refer: 
+- https://github.com/golang-migrate/migrate/blob/master/FAQ.md#what-does-dirty-database-mean
+
+### How are migrations applied? 
+
+- Whenever you need to make changes to the current applied schema, in the migrations folder, run `migrate create -ext sql -seq {{file_name}}`
+- A new version of a migration will be created. Once the changes are written and verified.
+- Make sure to write a rollback script in `__.down.sql` file. Otherwise the rollback of migrations won't be possible.
+- To apply the latest version of the migration, simply execute `go run migrate_db.go`
+- For additional utilities, for eg: drop the entire db, simply pass drop as a runtime argument like `go run migrate_db.go drop`
+- **DO NOT** drop `schema_migrations` table as that will lead to migration failure.
```

Comment:
I would also add on a suggestion to play more with markdowns, it will be helpful to beautify and make your writeups readable easily by the devs or end-user. Also, avoid using terms like `you/I` in the README(s). I have made some quick first mind suggestions that came to my mind but feel free to explore more. 
Try to add about the nomenclature of the migrateDB scaling files, also can add a point that these are autogenerated.

Reply:
Thanks for the suggestion, samyak. Will keep these in mind and hopefully can make better documents moving ahead!